lms/static/js/certificates/views/certificate_whitelist_editor.js

```diff
@@ -82,9 +85,9 @@
                     return re.test(email);
                 },
 
-                showMessage: function(message){
+                escapeAndShowMessage: function(message){
                     $(this.message_div +  ">p" ).remove();
-                    this.$(this.message_div).removeClass('hidden').append("<p>"+ gettext(message) + "</p>");
+                    this.$(this.message_div).removeClass('hidden').append("<p>"+ _.escape(message) + "</p>");
                 },
 
                 showSuccess: function(caller, add_model, message){
```

Comment:
@robrap I have verified that the input to `showSuccess` and `showError` is always strings and not html.